map_annotator/map_annotator/CMakeFiles/map_annotator_genlisp.dir/build.make

```diff
@@ -0,0 +1,72 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Avoid adding files like these to version control.

Usually people do this when they are in the habit of running `git add <whole directory>` or `git add -A`, and then `git commit -m ...`, which doesn't show the list of files in the commit. All of these should be avoided because make it too easy to make mistakes, and they encourage insufficient commit messages.

Reply:
I was not sure if CMakeFiles should be avoided when I committed the files. I will definitely avoid these in the future!

Reply:
Can I make the pull request after map annotation is done since I’m already committing files related to it in the branch?

Reply:
Ideally it'd be 1PR = 1 issue resolved/1 feature so let's try to get web teleop landed in the main branch as its own PR. I can show you how to handle this git wise before or after the meeting today